### DISCUSSION TITLE
DAG解决方案，连接线样式

### DISCUSSION THREAD

**Discussion by lishaopeng221** [antvis/XFlow#79] [Q&A] [answered]
1. DAG解决方案，节点增加左右连接点并连线后，连线箭头全部向下
![lALPJwnIwrkYU-3NARTNAqs_683_276](https://user-images.githubusercontent.com/25917470/148000828-720ad278-2e81-4b86-b8f7-730ebe826b4e.png)

2. DAG解决方案，开启历史选项以后，连接线箭头消失
![e16905dd2149a9960002aba66262b34](https://user-images.githubusercontent.com/25917470/148000787-7f3ea9e8-00f5-4941-ac0d-62b39d2c6b2f.png)

**Comment by lyn-boyu** [comment, marked as answer]
计划增加一个layoutDirection的配置项：'TB'｜'RL'  
修改配置项会同步修改一些样式
